pkgs/by-name/li/linien-client/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, python3
+, linien-common
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

Comment:
linien-client is also a library

Reply:
I think linien-client is an application that has the command linien.
https://github.com/linien-org/linien#installation-with-pip

Reply:
```
nix-build -A linien-client
/nix/store/4hwmg9xyvn9k7hr4ij66pva6pyc60425-linien-client-0.8.0


ls ./result/
lib  nix-support
```

Reply:
Indeed, they don't provide scripts.
https://github.com/search?q=repo%3Alinien-org%2Flinien+entry_points&type=code

Reply:
Will move client and common to python-modules and retain server and gui in pkgs/by-name